### ISSUE TITLE
World portals

### ISSUE THREAD

**Issue by Tjmcnee** [BEEmod/BEE2-items#2234]
Just like the in unreal chamber, they use world portals, but that's only in hammer, and I **SUCK**at hammer.
so can you make it in Bee2? Hopefully by the next release or the one after that, because I need these.

**Comment by BETTEREXTENEDEDITORMOD** [comment]
this is already going to be added

**Comment by Endermage77** [comment]
I regret to tell you, alas, that #95 aside (which is a very good catch-all post for stuff like this), that this has been brought up umpteen times already in a multitude of ways.

There's even a whole section I did suggesting some fancy shmancy new stuff the old one doesn't have (And probably still doesn't); @TeamSpen210 said in the comments on pr25 that the devs would refer back to that post on world portals, but as far as I can tell it's either taking a long while or they forgot about it.

See #1674 for the post I'm rambling on about.
See #1488 for a list of other spins on the standard door I suggested that have never existed but probably should.
See #942 for ellington's mid-chamber elevator idea that's essentially a fancy world-portal.
See #574 for a discussion about and not exactly about resizable and/or standing world portals and what is the third-longest comment chains in the BEE2 issues section (Standing behind panel arms and the aforementioned #95.)
See #306 for some very old discussion that is still floating about for some reason? Not sure if it's a duplicate considering nobody's caught it for like nearly two years.
